lua/lspconfig.lua

```diff
@@ -9,9 +9,26 @@ function M.available_servers()
   return M.util.available_servers()
 end
 
+---@class Alias
+---@field to string The new name of the server
+---@field version string The version that the alias will be removed in
+---@type table<string, Alias>
+local server_aliases = {
```

Comment:
this table can add a deprecate tag ? it should be remove when after 0.2.0 release right?

Reply:
All aliases in this table are deprecated, the version field is the version it will get removed, which for sumneko_lua is 0.2.0.

Reply:
I know what's the table mean. I mean like this. this table only for sumneko lua.

```lua
---@deprecate
local server_aliases
```